stable_baselines/common/base_class.py

```diff
@@ -198,23 +198,29 @@ def load(cls, load_path, env=None, **kwargs):
 
     @staticmethod
     def _save_to_file(save_path, data=None, params=None):
-        _, ext = os.path.splitext(save_path)
-        if ext == "":
-            save_path += ".pkl"
+        if isinstance(save_path, str):
+            _, ext = os.path.splitext(save_path)
+            if ext == "":
+                save_path += ".pkl"
 
-        with open(save_path, "wb") as file:
-            cloudpickle.dump((data, params), file)
+            with open(save_path, "wb") as file:
```

Comment:
```suggestion
            with open(save_path, "wb") as file_:
```

`file` is a python keyword. (I know that was there before but I think that is the right moment to change that ;) )